crates/ty_python_semantic/resources/mdtest/overloads.md

```diff
@@ -598,6 +598,7 @@ class CheckClassMethod:
     # error: [invalid-overload]
     def try_from3(cls, x: int | str) -> CheckClassMethod | None:
         if isinstance(x, int):
+            # error: [call-non-callable] "Object of type `CheckClassMethod` is not callable"
```

Comment:
nit: Since there's a snapshot test, consider not asserting the text of the error message. That makes less churn on the snapshot and means you're not double-checking the error message.